core/modules/dashboard/includes/block.menus.inc

```diff
@@ -0,0 +1,140 @@
+<?php
+/**
+ * @file
+ *
+ * Dashboard block displaying menus on the site.
+ */
+class DashboardMenuBlock extends Block {
+
+  /**
+   *  Sets block subject on block view.
+   */
+  function getTitle() {
+    return !empty($this->settings['title']) ? check_plain($this->settings['title']) : t('Menus');
+  }
+
+  /**
+   * 'Admin title' callback for the content type.
+   */
+  function getAdminTitle() {
+    return t('Manage Menus');
+  }
+
+  /**
+   * 'Admin info' callback for the content type.
+   */
+  function getAdminPreview() {
+    return t('Links to administer menus.');
+  }
+
+  /**
+   * Run-time rendering of the body of the block.
+   */
+  function getContent() {
+    if (!module_exists('menu')) {
+      return;
+    }
+    $items = array();
+
+    $menus = menu_get_menus();
+    $options = array('destination' => 'admin/dashboard');
+    $header = array(
+      array('data' => t('Menu')),
+      array(
+        'data' => t('Operations'),
+      ),
+    );
+    $rows = array();
+
+    if (user_access('administer menu')) {
+      foreach ($menus as $menu_name => $menu) {
+        if (array_key_exists($menu_name, $this->settings['menus'])) {
+          if ($this->settings['menus'][$menu_name] === $menu_name) {
+            $links['configure'] = array(
+              'title' => t('Configure'),
+              'href' => 'admin/structure/menu/manage/' . $menu_name . '/edit',
+              'query' => $options,
+            );
+            $links['manage'] = array(
+              'title' => t('Manage links'),
+              'href' => 'admin/structure/menu/manage/' . $menu_name,
+              'query' => $options,
+            );
+            $links['add'] = array(
+              'title' => t('Add new link'),
+              'href' => 'admin/structure/menu/manage/' . $menu_name . '/add',
+              'query' => $options,
+            );
+            $operations = array(
+              '#type' => 'dropbutton',
+              '#links' => $links,
+            );
+            $rows[] = array(
+              'data' => array(
+                t($menu),
```

Comment:
Hmm, have never seen `t()` being used with only a single variable in it. What the purpose of this?

Reply:
to translate the name of the menu. Is that a thing? I'll check the menu module...

Reply:
nope, switching to check_plain().

Reply:
Thought so 😄 👍